presto-main/src/main/java/com/facebook/presto/memory/MemoryResource.java

```diff
@@ -48,4 +50,12 @@ public MemoryInfo getMemoryInfo(MemoryPoolAssignmentsRequest request)
         taskManager.updateMemoryPoolAssignments(request);
         return memoryManager.getInfo();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public MemoryInfo getMemoryInfo()
+    {
+        taskManager.updateMemoryPoolAssignments(new MemoryPoolAssignmentsRequest("", 0, new ArrayList<>()));
```

Comment:
This method call does return current memory allocation on a worker. I agree with the part that request is empty, so is the goal to just have GET call with the request or we should expect the GET request parameters and use them to build request?
This is what a sample output looks like(Same as POST request)
```
{
  "totalNodeMemory" : "1288490189B",
  "pools" : {
    "reserved" : {
      "maxBytes" : 214748364,
      "freeBytes" : 214748364,
      "queryMemoryReservations" : { }
    },
    "general" : {
      "maxBytes" : 1073741825,
      "freeBytes" : 1073741825,
      "queryMemoryReservations" : { }
    },
    "system" : {
      "maxBytes" : 858993459,
      "freeBytes" : 858993459,
      "queryMemoryReservations" : { }
    }
  }
}
```